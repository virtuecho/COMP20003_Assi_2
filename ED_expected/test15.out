Parkville -->
COMP20003 Code10331, Official Code Suburb22038, Official Name SuburbParkville, Year2021, Official Code State2, Official Name StateVictoria, Official Code Local Government Area24600, 25060, 25250, Official Name Local Government AreaMelbourne, Moonee Valley, Moreland, Latitude-37.7866675, Longitude144.9512266
Far Far Away -->
Kensington -->
COMP20003 Code1117, Official Code Suburb21327, Official Name SuburbKensington, Year2021, Official Code State2, Official Name StateVictoria, Official Code Local Government Area24330, 24600, 25060, Official Name Local Government AreaMaribyrnong, Melbourne, Moonee Valley, Latitude-37.7943466, Longitude144.9270189
South -->
North Melbourne -->
COMP20003 Code4377, Official Code Suburb21966, Official Name SuburbNorth Melbourne, Year2021, Official Code State2, Official Name StateVictoria, Official Code Local Government Area24600, Official Name Local Government AreaMelbourne, Latitude-37.7984833, Longitude144.9448560
South Yarra -->
COMP20003 Code5004, Official Code Suburb22314, Official Name SuburbSouth Yarra, Year2021, Official Code State2, Official Name StateVictoria, Official Code Local Government Area24600, 26350, 27350, Official Name Local Government AreaMelbourne, Stonnington, Yarra, Latitude-37.8392817, Longitude144.9919139
Port Melbourne -->
COMP20003 Code5388, Official Code Suburb22107, Official Name SuburbPort Melbourne, Year2021, Official Code State2, Official Name StateVictoria, Official Code Local Government Area23110, 24330, 24600, 25900, Official Name Local Government AreaHobsons Bay, Maribyrnong, Melbourne, Port Phillip, Latitude-37.8332725, Longitude144.9222073
Buckingham -->
Melbourne -->
COMP20003 Code6623, Official Code Suburb21640, Official Name SuburbMelbourne, Year2021, Official Code State2, Official Name StateVictoria, Official Code Local Government Area24600, 25900, 26350, 27350, Official Name Local Government AreaMelbourne, Port Phillip, Stonnington, Yarra, Latitude-37.8249613, Longitude144.9715278
Carlton -->
COMP20003 Code9773, Official Code Suburb20495, Official Name SuburbCarlton, Year2021, Official Code State2, Official Name StateVictoria, Official Code Local Government Area24600, Official Name Local Government AreaMelbourne, Latitude-37.8004392, Longitude144.9680900
