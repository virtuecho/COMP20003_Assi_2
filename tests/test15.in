Parkville
Far Far Away
Kensington
South
North Melbourne
South Yarra
Port Melbourne
Buckingham
Melbourne
Carlton
