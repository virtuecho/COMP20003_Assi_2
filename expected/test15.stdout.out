Parkville --> 1 records found
Far Far Away --> NOT FOUND
Kensington --> 1 records found
South --> NOT FOUND
North Melbourne --> 1 records found
South Yarra --> 1 records found
Port Melbourne --> 1 records found
Buckingham --> NOT FOUND
Melbourne --> 1 records found
Carlton --> 1 records found
